packages/@webex/internal-plugin-dss/src/config.ts

```diff
@@ -4,6 +4,12 @@
 
 export default {
   dss: {
+    /**
+     * Timeout before DDS request fails.
+     * @type {Number}
+     */
+    requestTimeout: 6000,
```

Comment:
Did you ask DSS team about this?

Reply:
@lismith2-cisco I got it from the KMS plugin

Reply:
I've added you to DSS comms space and asked Willie

Reply:
Looks like maximum request time is around 100ms. I set the default timeout to `1000` it give us enough room for network slowness and it should not be noticeable for the user. Let me know if you disagree.